frontend/src/services/error-tracking.js

```diff
@@ -0,0 +1,44 @@
+import {
+    BrowserTracing,
+    Replay,
+    init,
+    vueRouterInstrumentation
+
+} from '@sentry/vue'
```

Comment:
Ideally, this would be loaded async so it's not required if sentry is not enabled (so there's no bundle size impact), unfortunately, Sentry needs to init before `Vue` for the hooks to work properly.

If we keep sentry around, we can look into including the SDK as part of the injected code in forge/routes/ui/index.js